src/wui/building_statistics_menu.cc

```diff
@@ -616,6 +699,29 @@ void BuildingStatisticsMenu::jump_building(JumpTarget target, bool reverse) {
 	update();
 }
 
+void BuildingStatisticsMenu::jump_traffic(JumpTarget target, bool reverse) {
```

Comment:
I get "unused parameter: target" compiler warning here.

Reply:
It's in the CI log too in release builds. Sorry, I don't know how to link to a line in the logs.